src/schemas.test.ts

```diff
@@ -13,4 +16,41 @@ describe('Schema validation', () => {
       })
     })
   })
+  it.each(
```

Comment:
quoting from the PR description:

> for schema validation I just relied on TS and wrote my own logic in jest tests rather than using Joi. I think Joi (or zod, or whatever) is overkill given that all the data is static. The schema validation libraries seem better for data you get dynamically, like request parameters, rather than data you already have on hand and just need to fit a certain shape.

Reply:
I agree it would be nice to get rid of Joi in this repo.

But even if it's static data, there are things that were important to check for tokens written in the RTDB. Or it could possibly crash clients which read directly from it.

The jest tests added here don't enforce as much. For instance if there's an additional property added by mistake to a token, it won't complain.

Which is probably fine I guess for now for ethereum tokens since they aren't synced to the RTDB, but since the repo is public and accepts contributions for new tokens, we should be sure the data is strictly in the right format.

Joi is stricter, though I don't really care what we use, as long as we enforce the strict format.

Reply:
Ok it sounds like we agree on a high level that Joi could be replaced without compromising data quality (e.g. by moving a test step into the "yarn update:rtdb" script), but it also sounds like you're suggesting punting on that for now, since it's not really necessary for adding ethereum token data and the new schema.

I'm going to try that out by making modest updates to the Joi schema to work for all cases and see if we can scrape by for now.